### PR TITLE
TK 4 Completed

### PR DESCRIPTION
**Pull request by johangv**
Terminado :3

### CODE REVIEW

File: src/Pokeview/components/Type.scss [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        box-shadow: 0px 4px 0 #A94FB5;
+    }
+
+    &-ground {
+        @include type-content();
+        background-color: #f2863e;
+        box-shadow: 0px 4px 0 #E27D3A;
+    }
+
+    &-rock {
```
Comment (bruno9926) [inline src/Pokeview/components/Type.scss:46]: the box shadow color is the right one?

File: src/Pokeview/components/Type.scss [hunk 2]
```diff
@@ -0,0 +1,122 @@
+@mixin type-content {
+    height: 60%;
```
Comment (bruno9926) [inline src/Pokeview/components/Type.scss]: I recommend you to use em instead of percentage units

File: src/Pokeview/components/Type.scss [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 0.5em;
+    font-size: 1.1vw;
```
Comment (bruno9926) [inline src/Pokeview/components/Type.scss]: I recommend you to use em instead of percentage units

File: src/Pokeview/components/Type.jsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './Type.scss';
+
+const Type = ({name, type}) => {
+  return (
+    <div className={(`j-type-${type}`)}>{name}</div>
```
Comment (bruno9926) [inline src/Pokeview/components/Type.jsx]: parenthesis are not necessary

File: src/Pokeview/components/Type.jsx [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './Type.scss';
+
+const Type = ({name, type}) => {
```
Comment (bruno9926) [inline src/Pokeview/components/Type.jsx]: name is also not necessary, you can use the type prop, it doesn't matter if it is in english

File: src/Pokeview/Pokeview.scss [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+        bottom: 13%;
+
+        @media screen and (max-width: 773px) {
+            width: 30%;
+            left: 10%;
+        }
+
+    }
+
+    &-container > div {
```
Comment (bruno9926) [inline src/Pokeview/Pokeview.scss]: I encourage you to use a class instead of a simple div selector

File: src/Pokeview/Pokeview.scss [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+        position: absolute;
+        bottom: -7%;
+        left: 8%;
+        gap: 0.5vw;
```
Comment (bruno9926) [inline src/Pokeview/Pokeview.scss]: use em instead of vw

File: src/Pokeview/Pokeview.scss [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+        width: 20vw;
```
Comment (bruno9926) [inline src/Pokeview/Pokeview.scss]: same

File: src/Pokeview/Pokeview.scss [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+        height: 13%;
```
Comment (bruno9926) [inline src/Pokeview/Pokeview.scss]: use em instead

File: src/App.js [hunk 10]
```diff
@@ -13,6 +14,7 @@ function App() {
         <div className="App bg" >
           <Navbar></Navbar>
           <Home />
```
Comment (bruno9926) [inline src/App.js]: Put the pokeview component inside of the Home page

File: src/pages/Home/Home.jsx [hunk 11]
```diff
@@ -6,7 +6,7 @@ import Welcome from '../../Welcome/Welcome';
 const Home = () => {
   return (
     <div className='home-page'>
-        <Welcome />
+        {/* <Welcome /> */}
```
Comment (bruno9926) [inline src/pages/Home/Home.jsx:11]: put pokeview component here

**Review by bruno9926 — CHANGES_REQUESTED** [review]
Great job! resolve the little issues and you are ready to go

File: src/Pokeview/components/Type.jsx (hunk 4 above)
Comment (johangv) [reply src/Pokeview/components/Type.jsx]: Fixed

File: src/Pokeview/components/Type.jsx (hunk 5 above)
Comment (johangv) [reply src/Pokeview/components/Type.jsx]: Fixed

File: src/Pokeview/components/Type.scss (hunk 2 above)
Comment (johangv) [reply src/Pokeview/components/Type.scss]: Fixed

File: src/Pokeview/components/Type.scss (hunk 3 above)
Comment (johangv) [reply src/Pokeview/components/Type.scss]: fixed

File: src/Pokeview/components/Type.scss (hunk 1 above)
Comment (johangv) [reply src/Pokeview/components/Type.scss:46]: Fixed

File: src/Pokeview/Pokeview.scss (hunk 7 above)
Comment (johangv) [reply src/Pokeview/Pokeview.scss]: Fixed

File: src/Pokeview/Pokeview.scss (hunk 6 above)
Comment (johangv) [reply src/Pokeview/Pokeview.scss]: Fixed

File: src/Pokeview/Pokeview.scss (hunk 8 above)
Comment (johangv) [reply src/Pokeview/Pokeview.scss]: Fixed

File: src/Pokeview/Pokeview.scss (hunk 9 above)
Comment (johangv) [reply src/Pokeview/Pokeview.scss]: Fixed

File: src/pages/Home/Home.jsx (hunk 11 above)
Comment (johangv) [reply src/pages/Home/Home.jsx:11]: Done

File: src/App.js (hunk 10 above)
Comment (johangv) [reply src/App.js]: Done

**Comment by johangv** [comment]
All comments was fixed

**Pull request merged**